src/main.cpp

```diff
@@ -14,19 +15,24 @@ int main()
     freopen("CON", "r", stdin);
     #endif
 
+    static const constexpr int WIDTH = 800;
+    static const constexpr int HEIGHT = 600;
```

Comment:
can be put at top of file or in renderer class or default values or whatever you know dude